platform/viewer/src/routes/ViewerRouting.js

```diff
@@ -36,7 +36,7 @@ function ViewerRouting({ match: routeMatch, location: routeLocation }) {
     return (
       <ConnectedViewerRetrieveStudyData
         studyInstanceUids={studyUIDs}
-        seriesInstanceUids={seriesUIDs}
+        seriesInstanceUIDs={seriesUIDs}
```

Comment:
Why the change in casing? I feel like we more consistently use `Uids`.

Reply:
I spoke to Erik and he mentioned UID would be more appropriate.
But again.. I see places with different versions.. I`m not that confident using one or other..
totally opened to implement what you think is the best one

Reply:
I don't have a strong preference, I just want us to be consistent. I'll poke Erik tomorrow.

Reply:
I will change it to seriesInstanceUids!